.github/workflows/haskell.yml

```diff
@@ -18,8 +18,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest]
         compiler-nix-name: [ghc810, ghc92, ghc96]
+        include:
+          # We want a single job, because macOS runners are scarce.
+          - os: macos-latest
+            compiler-nix-name: ghc810
```

Comment:
Don't we want to default to 9.6 here? We want to decommission 8.10 asap. 8.10 is only around for backup in case 9.6 somehow terribly doesnt' work out.

Reply:
OK, I updated to ghc810 for macOS, and left 810 and 96 for linux. Hopefully I can drop 810 soon